ambari-web/app/mappers/components_state_mapper.js

```diff
@@ -103,7 +103,8 @@ App.componentsStateMapper = App.QuickDataMapper.create({
       ranger_tagsyncs_total: 'TOTAL_PATH'
     },
     'MAPREDUCE2_CLIENT': {
```

Comment:
Will it be better if we remove the whole config of "MAPREDUCE2_CLIENT" and `map_reduce2_clients` here https://github.com/apache/ambari/blob/trunk/ambari-web/app/mappers/service_metrics_mapper.js#L137 ?

Reply:
@kevinw66 Thanks for your review.
It works fine after removing this code and I find the variable  `mapReduce2Client` never be used in ambari web.
So it is safe to delete this code